servicebox-app/src/main/java/io/halkyon/services/ClaimStatus.java

```diff
@@ -6,11 +6,9 @@ public enum ClaimStatus {
     // When claim is waiting for a service to be available
     PENDING,
     // When claim has a linked service
-    CLAIMED,
```

Comment:
Could you keep the original status and include them within a block comment ?
Can you document all the status to explain when we move from status A to B, etc ?